### ISSUE TITLE
Next Steps Depending on data entered on previous steps

### ISSUE THREAD

**Issue by jayomayan** [vildanbina/livewire-wizard#13]
This is more of a question, is it possible to show fields based on the value entered from the previous steps?

**Comment by trippo** [comment]
Yes but not easily you can access to state with `$container->getLivewire()->state`
